docs/source/docker/known_issues_faq.rst

```diff
@@ -56,16 +56,38 @@ workers for data fetching `lightly.loader.num_workers` there might be not enough
         fd, size = storage._share_fd_()                                                                                                                                                                         
     RuntimeError: unable to write to file </torch_31_1030151126> 
 
-To solve this problem we need to increase the shared memory for the docker runtime.
+To solve this problem we need to reduce the number of workers or 
+increase the shared memory for the docker runtime. 
 
-You can change the shared memory to 512 MBytes by adding `--shm-size="512m"` to 
-the docker run command:
+Lightly determines the number of CPU cores available and sets the number
+of workers to the same number. If you have a machine with many cores but not so much
+memory (e.g. less than 2 GB of memory per core) it can happen that you run out 
+of memory and you rather want to reduce
+the number of workers intead of increasing the shared memory. 
+
+You can set the number of workers using
+
+.. code-block:: console
+
+    # you can manually override the config to only use 4 workers
```

Comment:
this will actually be overwritten by the config from the api so I'd leave this away or link to the docs on the num_workers

Reply:
Good point!